### PR TITLE
create-readme

### PR DESCRIPTION
**Pull request by vvenance**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# OpenAPI & AsyncAPI example files
+
+This directory holds [OpenAPI](https://spec.openapis.org/oas/latest.html) & [AsyncAPI](https://www.asyncapi.com/docs/reference/specification/latest) example API Contracts files for [standalone APIs](https://github.com/bump-sh/examples/tree/main/apis) and [API Hubs](https://github.com/bump-sh/examples/tree/main/hubs) which are deployed to [bump.sh](https://bump.sh) to generate elegant and always up-to-date API documentation.
```
Comment (paulRbr) [inline README.md]:
```suggestion
This directory holds [OpenAPI](https://spec.openapis.org/oas/latest.html) & [AsyncAPI](https://www.asyncapi.com/docs/reference/specification/latest) example API contract files for [standalone APIs](https://github.com/bump-sh/examples/tree/main/apis) and [API Hubs](https://github.com/bump-sh/examples/tree/main/hubs) which are deployed to [bump.sh](https://bump.sh) to generate elegant and always up-to-date API documentation.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## View example standalone API contract files and generated documentation
+
+You can find all available examples of OpenAPI and AsyncAPI API contracts files in the [`api` folder of this repo](https://github.com/bump-sh/examples/tree/main/apis)
```
Comment (paulRbr) [inline README.md]:
```suggestion
You can find all available examples of OpenAPI and AsyncAPI API contract files in the [`apis` folder of this repo](https://github.com/bump-sh/examples/tree/main/apis)
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## View example API contract files for API Hubs and generated documentation
```
Comment (paulRbr) [inline README.md]:
```suggestion
## View example API Hubs and generated documentation
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+You can find all available examples of Hubs of OpenAPI and AsyncAPI API contracts files in the [`hubs` folder of this repo](https://github.com/bump-sh/examples/tree/main/hubs)
```
Comment (paulRbr) [inline README.md]:
```suggestion
You can find all available examples of Hubs (or API Portals) of OpenAPI and AsyncAPI API contract files in the [`hubs` folder of this repo](https://github.com/bump-sh/examples/tree/main/hubs)
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## What are these API contract files ?
+
+Most of them are publicly available API contract files from real tech companies and products.
+Bump created and owned API contract files are:
+- [Bump ](https://github.com/bump-sh/examples/tree/main/hubs/my-train-company) is Bump.sh [publicly available demo](https://demo.bump.sh/)
```
Comment (paulRbr) [inline README.md]:
```suggestion
- [Bump](https://github.com/bump-sh/examples/tree/main/apis/bump.yml) is Bump.sh [publicly available API](https://developers.bump.sh)
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+### I want to add a **new** standalone API specification file
+
+3️⃣ Create a new file under `./apis/`. Follow this file naming convention:
+
+```
+<bump-doc-slug>-[openapi|asyncapi]-source.[json|yml]
+```
+
+`bump-doc-slug` must be replaced by your file name.
```
Comment (paulRbr) [inline README.md]: Not sure what “by your file name” means? Should we mention “API name”?

File: README.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+4️⃣ `git add <path/to/file>` your new API specification file content and `git commit` the file to your new branch.
+5️⃣ `git push origin <branch-name>` from your new branch and create a new pull request in this repo.
+6️⃣ In the pull request description add a link to the public source of the file.
+
+
+### I want to update standalone API or Hubs specification files
+
+ℹ️ The only edits allowed are updates of the public specification files to reflect more up-to dates versions of the API contracts.
+
+3️⃣ Locate the file you would like to edit under `./apis/` if Standalone API or repository under `./hubs` if an API portal and proceed with updating the documents.
```
Comment (paulRbr) [inline README.md]:
```suggestion
3️⃣ Locate the file you would like to edit under `./apis/` if Standalone API or under `./hubs` if an API portal and proceed with updating the documents.
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+4️⃣ `git add <path/to/file>` the updated API specification files and `git commit` the files to your new branch.
+5️⃣ `git push origin <branch-name>` from your new branch and create a new pull request in this repo.
+6️⃣ In the pull request description add a link to the public source of the files.
+
+
+### Note to maintainers on new APIs specification files
+
+In order for the files to be deployed automatically, we must create a new Documentation or Hub in the Bump demo organization.
```
Comment (paulRbr) [inline README.md]:
```suggestion
In order for the files to be deployed automatically, we must create a new Documentation or Hub in the [Bump demo organization](https://bump.sh/demo/docs).
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+## View example standalone API contract files and generated documentation
+
+You can find all available examples of OpenAPI and AsyncAPI API contracts files in the [`api` folder of this repo](https://github.com/bump-sh/examples/tree/main/apis)
+
+## View example API contract files for API Hubs and generated documentation
+
+You can find all available examples of Hubs of OpenAPI and AsyncAPI API contracts files in the [`hubs` folder of this repo](https://github.com/bump-sh/examples/tree/main/hubs)
+
+## What are these API contract files ?
```
Comment (paulRbr) [inline README.md]:
```suggestion
## What are these API contract files?
```

File: README.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+
+Most of them are publicly available API contract files from real tech companies and products.
+Bump created and owned API contract files are:
+- [Bump ](https://github.com/bump-sh/examples/tree/main/hubs/my-train-company) is Bump.sh [publicly available demo](https://demo.bump.sh/)
+- [My train Company](https://github.com/bump-sh/examples/tree/main/hubs/my-train-company) is Bump.sh publicly available demo and has its live generated documentation [available here](https://demo.bump.sh/).
+
+## How to contribute ?
```
Comment (paulRbr) [inline README.md]:
```suggestion
## How to contribute?
```

File: README.md (hunk 6 above)
Comment (vvenance) [reply README.md]: editing

**Comment by vvenance** [comment]
@paulRbr all changes have been implemented, do you approve now ?

**Comment by vvenance** [comment]
ℹ️ must be squashed before merge

**Review by paulRbr — APPROVED** [review]
purfect, many thanks for this README

**Pull request merged**